edsc/plugins/datasource/cwic/src/js/Granule.jsx

```diff
@@ -1,12 +1,39 @@
 import extend from './extend.jsx';
+import * as XmlHelpers from './XmlHelpers.jsx';
+import * as CwicUtils from './CwicUtils.jsx';
+
+let ajax = window.edsc.util.xhr.ajax;
 
 let CwicGranule = (function() {
   extend(CwicGranule, window.edsc.models.data.Granule);
 
   function CwicGranule(jsonData) {
     CwicGranule.__super__.constructor.apply(this, Array.prototype.slice.call(arguments));
+    this.details = this.asyncComputed({}, 100, this._computeGranuleDetails, this);
```

Comment:
I don't think you need this. Rather, you shouldn't nee this, so please find a way get rid of it. Inheritance should cause the correct method to be used.